sunset/code/modules/mob/dead/observer/observer.dm

```diff
@@ -0,0 +1,20 @@
+/mob/living/ghost()
```

Comment:
you should still be able to ghost without cryopods

Reply:
Yes, that's why I said a hook.

check_cryopod would only `return FALSE` if the player was in a cryopod and clicked no, otherwise would default to `return TRUE`

Reply:
don't override the whole damn proc when a simple 2-liner can be added to the file.